api/build.gradle

```diff
@@ -11,13 +11,11 @@ description = 'gRPC: API'
 evaluationDependsOn(project(':grpc-context').path)
 
 dependencies {
-    compile project(':grpc-context'),
-            libraries.errorprone,
-            libraries.jsr305,
-            libraries.animalsniffer_annotations
-    guavaDependency 'compile'
+    api project(':grpc-context'),
+            libraries.jsr305
```

Comment:
Why expose this one?

Reply:
It is part of public API signature.

Reply:
I mean annotations like `@javax.annotation.concurrent.ThreadSafe`, `@javax.annotation.Nullable` are part of some public API signature.